app/controllers/feedback_messages_controller.rb

```diff
@@ -20,6 +20,13 @@ def create
       )
       rate_limiter.track_limit_by_action(:feedback_message_creation)
 
+      if user_signed_in?
+        Rails.cache.fetch("user-#{current_user.id}-feedback-response-sent-at", expires_in: 24.hours) do
```

Comment:
I don't like that the cache is written not at the actual time when an email is sent but at the time the job is scheduled.
An alternate solution would be to create a separate worker and fetch cache there. But in that case, we either check cache twice (in the worker and in the controller) or schedule redundant workers.
I decided that in this case creating a worker would be overkill. In this case, I write time to the cache just in case, we don't need the actual time, but the record presence.